### PR TITLE
Add .NET v2 client alpha release post

### PR DESCRIPTION
**Pull request by bruth**

### CODE REVIEW

File: content/blog/nats-dotnet-v2-alpha-release.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+await foreach (var msg in sub.Msgs.ReadAllAsync())
+{
+    Console.WriteLine($"Received request: {msg.Data}");
+
+    // Reply to the sender of the message via the reply subject
+    await msg.ReplyAsync($"Answer is: { 2 * msg.Data }");
+}
+
+:
```
Comment (caleblloyd) [inline content/blog/nats-dotnet-v2-alpha-release.md:66]:
```suggestion
```

Extra character snuck in

**Review by caleblloyd — APPROVED** [review]
Looks great!

File: content/blog/nats-dotnet-v2-alpha-release.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+The original NATS .NET client, called [NATS.Client v1](https://www.nuget.org/packages/NATS.Client) on NuGet, pre-dates these advancements and Microsoft’s new LTS roadmap - it was originally released in 2015! Backwards compatibility remains a large commitment in NATS.Client v1, which would make it difficult to port in order to take advantage of new APIs.
+
+NATS community members started to take note, and develop client libraries for NATS based on modern .NET APIs. One notable client library that emerged was the [AlterNats](https://github.com/Cysharp/AlterNats) library by Cysharp, which includes a fully asynchronous API, leverages [Span&lt;T&rt;](https://learn.microsoft.com/en-us/archive/msdn-magazine/2018/january/csharp-all-about-span-exploring-a-new-net-mainstay), and supports client-side WebSockets from browsers in [Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor). NATS maintainers and AlterNats maintainers agreed that AlterNats would make a great starting point for NATS.Client v2!
```
Comment (mtmk) [inline content/blog/nats-dotnet-v2-alpha-release.md:14]: Should this be `&gt;` in `Span&lt;T&rt;`?

Comment (bruth) [reply content/blog/nats-dotnet-v2-alpha-release.md:14]: Bah, good catch.

File: content/blog/nats-dotnet-v2-alpha-release.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
```
Comment (bruth) [inline content/blog/nats-dotnet-v2-alpha-release.md:14]:
```suggestion
NATS community members started to take note, and develop client libraries for NATS based on modern .NET APIs. One notable client library that emerged was the [AlterNats](https://github.com/Cysharp/AlterNats) library by Cysharp, which includes a fully asynchronous API, leverages [Span&lt;T&gt;](https://learn.microsoft.com/en-us/archive/msdn-magazine/2018/january/csharp-all-about-span-exploring-a-new-net-mainstay), and supports client-side WebSockets from browsers in [Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor). NATS maintainers and AlterNats maintainers agreed that AlterNats would make a great starting point for NATS.Client v2!
```

**Review by gcolliso — APPROVED** [review]
lgtm

File: content/blog/nats-dotnet-v2-alpha-release.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+Today, we’re excited to announce our first developer preview release of [NATS.Client.Core v2](https://www.nuget.org/packages/NATS.Client.Core), which supports [Core NATS](https://docs.nats.io/nats-concepts/core-nats). JetStream support is under development and will be coming soon!
+
+## What’s Supported?
+
+[NATS.Client.Core v2](https://www.nuget.org/packages/NATS.Client.Core) is a complete implementation of all Core NATS features including headers. We take full advantage of async/await, IAsyncDisposable, Channels, IAsyncEnumerable and ValueTask to provide a great developer experience and high performance.
+
+### Publish-subscribe
```
Comment (bruth) [inline content/blog/nats-dotnet-v2-alpha-release.md:22]:
```suggestion
### Publish-Subscribe
```

**Pull request merged**